rust/rust_sketch/build.rs

```diff
@@ -13,6 +13,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+use cargo_metadata::MetadataCommand;
+use serde_json::Value;
+
+pub const DEFAULT_TAG: &str = "5.2.0";
```

Comment:
It doesn't look like there's an obvious way to know whether this is used or the version in the Cargo.toml. Maybe we should remove this, and just panic if we can't get the version from the Cargo.toml?